pkg/apis/core/validation/shoot_test.go

```diff
@@ -643,17 +630,6 @@ var _ = Describe("Shoot Validation Tests", func() {
 				))
 			})
 
-			It("should enforce workers min > 0 if max > 0", func() {
```

Comment:
Can you invert this test case, instead of removing it?
I guess, it makes sense to have a test case for checking, that `min=0` is allowed.

Reply:
Sure, I can do that.

Reply:
done